lib/fog/ovirt/requests/compute/v4/list_vm_volumes.rb

```diff
@@ -3,18 +3,25 @@ module Compute
     class Ovirt
       class V4
         class Real
+          # rubocop:disable Metrics/AbcSize
           def list_vm_volumes(vm_id)
             # disk_attachments_service = client.system_service.vms_service.vm_service(vm_id).disk_attachments_service
             # disk_attachments_service.list.map {|ovirt_obj| ovirt_attrs client.system_service.disks_service.disk_service(ovirt_obj.id).get}
```

Comment:
These two lines look like a leftover. Is there any value in keeping them around?